main.css

```diff
@@ -0,0 +1,4 @@
+.about-header {
```

Comment:
can we rename this to title or header?  then we can reuse it for all headers that need to be centered